babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -209,6 +209,17 @@ SqlNode SqlExecMacro() :
     }
 }
 
+SqlNode SqlUsingRequestModifier(Span s) :
```

Comment:
Since this function is calling ExtendColumnList(), base on some good practice in ordering function, I suggest this function could be move upward in the code so the function call dependency would point in a downward direction.

Reply:
Good suggestion, I'll keep that in mind for future features :)